packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-form.jsx

```diff
@@ -0,0 +1,236 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Autocomplete,
+    Button,
+    DesktopWrapper,
+    MobileWrapper,
+    SelectNative,
+    FormSubmitErrorMessage,
+    Text,
+    Timeline,
+} from '@deriv/components';
+import { Formik, Field } from 'formik';
+import FormSubHeader from 'Components/form-sub-header';
+import { localize, Localize } from '@deriv/translations';
+import { isDesktop, WS } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+import FormFooter from 'Components/form-footer';
+import FormBody from 'Components/form-body';
+import LoadErrorMessage from 'Components/load-error-message';
+import PoincFileUploaderContainer from 'Components/poinc/file-uploader-container';
+
+let file_uploader_ref = null;
+
+const ProofOfIncomeForm = ({
+    addNotificationByKey,
+    removeNotificationByKey,
+    removeNotificationMessage,
+    poinc_documents_list,
+    onSubmit,
+}) => {
+    const [document_file, setDocumentFile] = React.useState({ files: [], error_message: null });
+    const [disabled_items, setDisabledItems] = React.useState([2]);
+    const [api_initial_load_error, setAPIInitialLoadError] = React.useState(null);
+    const [uploading_document_type, setUploadingDocumentType] = React.useState('');
+
+    const initial_form_values = {
+        document_type: '',
+    };
+
+    const validateFields = values => {
+        const errors = {};
+        const { document_type } = values;
+
+        if (!document_type) {
+            errors.document_type = localize('This field is required.');
+        } else if (!poinc_documents_list.find(c => c.text === document_type)) {
+            errors.document_type = localize('This field is required.');
```

Comment:
Why do you create a nested condition if the error message is the same?

Reply:
thanks for this, I will refactor this part